articles/fleetctl.md

```diff
@@ -10,25 +10,16 @@ fleetctl also provides a quick way to work with all the data exposed by Fleet wi
 
 ## Installing fleetctl
 
-Download and install [Node.js](https://nodejs.org/en).
 
-Install fleetctl with npm (included in Node.js).
-
-```sh
-sudo npm install -g fleetctl
-```
-
-To install fleetctl on Windows or Linux, download the fleectl binary from [GitHub](https://github.com/fleetdm/fleet/releases).
+To install fleetctl on macOS, Windows, or Linux, download the fleectl binary from [GitHub](https://github.com/fleetdm/fleet/releases).
```

Comment:
How do I run fleetctl after I download it from GitHub?

The reason we have npm in here is b/c running fleetctl is intuitive after installing via npm: run `fleetctl`. 

But, if running it is intuitive (or we need some extra steps) via GitHub download then I'm all for that!

Reply:
> Most admins know how to run command line binaries

I hear you but what about someone totally new to CLI binaries? We want the guides to be that easy to understand.

> I can add some additional comments to the instructions if we think it's been oversimplified and isn't clear.

I think something like this is a great idea 👍 

We have a install commands on [fleetdm.com/try-fleet](https://fleetdm.com/try-fleet) (you have to be logged in) that might make the initial download + upgrades easier. Is there anyway we can use these commands in the guide? This way, we're not maintaining two different ways / sets of instructions.

Here's the one for macOS:
![Screenshot 2024-11-19 at 10 10 16 AM](https://github.com/user-attachments/assets/f0fb6324-c820-4672-94ad-c1071f5e0eb1)

Reply:
We're probably opening a can of worms here... 
That command would be a much easier way to approach this however, once installed, it displays information to the user about getting a local test environment set up so this would add its own confusion to prospects and customers:
```
echo "To start the local demo:"
echo
echo "1. Start Docker Desktop"
echo "2. To access your Fleet Premium trial, head to fleetdm.com/try-fleet and run the command in step 2."
```

It also mentions what the latest version of `npm` is but doesn't actually download `npm` or do anything with it. Kind of weird... 
```
# Fetch the latest version number from NPM
latest_strippedVersion=$(curl -s "https://registry.npmjs.org/fleetctl/latest" | grep -o '"version": *"[^"]*"' | cut -d'"' -f4)
echo "Latest version available on NPM: $latest_strippedVersion"
```

Looking at that script, all it is doing is the same steps I have proposed updating our documentation to say -- going to our GitHub repo and downloading the latest version of `fleetctl`. The script also does not add the `fleetctl` binary to the user's `$PATH` so it's even more confusing how to actually run it because it installs it to a hidden folder `$HOME/.fleetctl`. With the steps in our documentation, at least you know where it's getting installed. 

I'll add steps for how to use the binary after installing it via GitHub direct download.

Reply:
> That command would be a much easier way to approach this however, once installed, it displays information to the user about getting a local test environment set up so this would add its own confusion to prospects and customers

> The script also does not add the fleetctl binary to the user's $PATH so it's even more confusing how to actually run it because it installs it to a hidden folder $HOME/.fleetctl

Dang. Good points

> I'll add steps for how to use the binary after installing it via GitHub direct download.

Sounds good 👍